src/Modal/Modal.tsx

```diff
@@ -0,0 +1,48 @@
+import React, { FC } from 'react'
+import styled from 'styled-components'
+import { fromTheme, defaultStyledProps, StyledProps } from '../utils/styled'
+import { Theme } from '../themes'
+
+const ModalWrapper = styled.div`
+  position: absolute;
+  width: 600px;
+`
+
+const Header = styled.div`
```

Comment:
I'm creating a Heading component in #28, let's re-use that. I'm doing some more stuff on that branch, so if we merge this first, I'll integrate it in the Modal component.